src/routes/auth/index.js

```diff
@@ -0,0 +1,39 @@
+import express  from 'express';
+import cookieParser from 'cookie-parser';
+import session  from 'express-session';
+import SequelizeStoreFactory from 'connect-session-sequelize';
+
+import basic    from './basic';
+import facebook from './facebook';
+
+import config   from '../../config';
+import { sequelize }       from '../../models/db';
+
+const router = express.Router();
+
+router.use('/basic', basic);
```

Comment:
no session for the basic auth, that's why I use it before doing the session stuff